### PR TITLE
Adds the possibility to list a single account.

### PR DESCRIPTION
**Pull request by bruno-kakele**
Notice that it can list any account, even if not in the wallets.

Tested with go run list..

### CODE REVIEW

**Comment by bruno-kakele** [comment]
@hectorchu @richiedunk Please let me know if this is the correct way to contribute to gonano. Thank you!

File: cmd/list.go [hunk 1]
```diff
@@ -15,7 +15,12 @@ var listCmd = &cobra.Command{
 	Use:   "list",
 	Short: "List wallets or accounts within a wallet",
 	Run: func(cmd *cobra.Command, args []string) {
-		if walletIndex < 0 {
+		if walletAccount != "" {
+			// For when a specific account is specified. A single account is returned.
+			rpcClient := rpc.Client{URL: rpcURL}
```
Comment (hectorchu) [inline cmd/list.go:20]: Can this line be moved into getBalanceAndPrint?

Comment (bruno-kakele) [reply cmd/list.go]: Done!

Comment (bruno-kakele) [reply cmd/list.go]: @hectorchu by the way, do you have Discord? Would it be good to setup a Discord channel for gonano quick support and discussions when other people want to contribute?

Comment (hectorchu) [reply cmd/list.go]: I think it's better to keep it all in GitHub.

Comment (bruno-kakele) [reply cmd/list.go]: Ok, np. What is the best way to chat about new contributions to gonano?

Please check commit c33c60d

Comment (hectorchu) [reply cmd/list.go]: By opening a new issue.

**Comment by bruno-kakele** [comment]
Thank you Hector!

**Pull request merged**